repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -72,11 +72,11 @@ class ExecutorClassLoader(
   override def findClass(name: String): Class[_] = {
     userClassPathFirst match {
       case true => findClassLocally(name).getOrElse(parentLoader.loadClass(name))
-      case false => {
+      case false =>
```

Comment:
Personally, I think "match ... case true ... case false" is silly. You can make this a simple if-else